README.md

```diff
@@ -814,6 +814,21 @@ const MyConnectedComponent = connect(
 )(MyComponent);
 ```
 
+### Hot reloading
+
+Given a Found-enhanced Redux store, you can dynamically replace the route configuration with `found.replaceRouteConfig` on the store object. This will replace the route configuration and run the match with the current location against the new route configuration.
```

Comment:
It so happens (and the tests use this) that you can get to the store with `instance.store` when using `createFarceRouter`, but I don't think this is something that we want to explicitly document, since it's really just an accident of implementation.